packages/cli/src/commands/setup/auth/templates/dbAuth.function.ts.template

```diff
@@ -109,6 +109,8 @@ export const handler = async (
     //
     // If this returns anything else, it will be returned by the
     // `signUp()` function in the form of: `{ message: 'String here' }`.
+
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I don't think we ever came to a consensus about whether we should have comments like these in the templates or not.

One way to get around that discussion is to actually use the variable.

Is it a horrible idea to add something like this?

```js
// Extra user attributes will be returned in `userAttributes` if there are any
console.log('userAttributes', userAttributes)
```

I mean, I'd prefer to find a better use of `userAttributes`, but couldn't come up with anything.

Another option is to rewrite it like this:

```js
handler: ({
  username: email,
  hashedPassword,
  salt,
  userAttributes: _userAttributes
}) => {
  return db.user.create({
    data: { email, hashedPassword, salt, /* name: userAttributes.name */ },
  })
}
```

or

```js
handler: ({
  username: email,
  hashedPassword,
  salt,
  userAttributes: _userAttributes
}) => {
  // Any extra attributes for the user are available on the `userAttributes`
  // object like this: `userAttributes.name`
  return db.user.create({
    data: { email, hashedPassword, salt },
  })
}
```

(That comment text can most likely be improved)

Reply:
I frankly have no preference here – i can live with the way it currently is, but agree it can be improved. But definitely that is out of the initial scope of this PR, so maybe do that individually?

#scopeCreep ;)

Otherwise, i'd leave it to you if you want to change it in this PR and push another commit on top, feel free (in which case maybe also to the PR title).

Reply:
It was mainly an idea to get around having to decide about if we should have `// eslint-disable-next-line @typescript-eslint/no-unused-vars` comments or not. But now that I see how many others there already are, doing some kind of workaround just to get rid of this one doesn't help much...

I think we'll have to take this one up for discussion yet again...

Reply:
I don't feel it's a big problem, but i can see why this one is particularily ugly ;)

I was wondering: Shouldn't `// eslint-disable-next-line no-unused-vars` also work for the typescript linter?

One thing i can say is that i believe we should never do auto-fixing in the CI pipeline, but rather make it fail, so dev's can correctly fix it.

Reply:
Resolving for now, OK?